logging/list.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START list]
+var google = require('googleapis');
+var projectId = process.env.TEST_PROJECT_ID || '<your-project-id>';
+var logging = google.logging('v2beta1').entries;
```

Comment:
We have veneer support for this now - can we use that instead?
https://github.com/GoogleCloudPlatform/gcloud-node#google-cloud-logging-beta